parser/parser.go

```diff
@@ -349,6 +367,50 @@ func ParseLV(line string) (*LV, error) {
 	}, nil
 }
 
+func ParseVG(line string) (*VG, error) {
+	// vgs --units=b --separator="<:SEP:>" --nosuffix --noheadings -o vg_name,vg_size,vg_free,vg_uuid,vg_tags --nameprefixes -a
+	components := strings.Split(line, separator)
+	if len(components) != 5 {
+		return nil, fmt.Errorf("expected 5 components, got %d", len(components))
+	}
+
+	fields := map[string]string{}
```

Comment:
It'd be nice to generalize this part of the LV/VG parsers.

Reply:
Yup makes sense, I'll move that part into its own method.